routes/questions.js

```diff
@@ -46,68 +24,74 @@ questionRouter.get('/v1/questions/:id([0-9]{1,})', function (req, res) {
+  const invalidReq = reqBody.title === null || reqBody.title === '' || reqBody.title === undefined;
   if (invalidReq) {
     res.status(400);
     res.json({ message: 'Bad Request. Question must have a title.' });
   } else {
-    var newId = questions[questions.length - 1].id + 1;
-    var timeNow = new Date();
-    var timeStr = timeNow.toUTCString();
+    const newId = questions[questions.length - 1].id + 1;
+    const timeNow = new Date();
+    const timeStr = timeNow.toUTCString();
     questions.push({
       id: newId,
       title: reqBody.title,
       body: reqBody.body,
       timeSubmitted: timeStr,
       username: reqBody.username,
-      answers: []
+      answers: [],
+    });
+    res.status(201);
+    res.json({status: 'Successful',
+      message: `New question: ${reqBody.title} added.`,
+      location: `/v1/questions/${newId}`
     });
-    res.json({ message: 'New question: ' + reqBody.title + ' added.', location: '/v1/questions/' + newId });
   }
 });
 
+
 // POST AN ANSWER
-questionRouter.post('/v1/questions/:id([0-9]{1,})/answers', function (req, res) {
-  var reqBody = req.body;
-  var requestId = req.params.id;
-  var invalidReq = reqBody.body === null || reqBody.body === '' || reqBody.body === undefined;
+questionRouter.post('/v1/questions/:id([0-9]{1,})/answers', (req, res) => {
+  const reqBody = req.body;
+  const requestId = req.params.id;
+  const invalidReq = reqBody.body === null || reqBody.body === '' || reqBody.body === undefined;
 
   // Find the question with the request id.
-  var currQuestion = questions.filter(function (question) {
+  const currQuestion = questions.filter((question) => {
     if (question.id === parseInt(requestId)) {
       return true;
     }
   });
 
   if (currQuestion.length < 1) {
-    res.status(404); // Set status to 404 as question was not found
-    res.json({ message: 'Question ' + requestId + ' Not Found' });
+    res.status(404);// Set status to 404 as question was not found
+    res.json({ message: `Question ${requestId} Not Found` });
   } else if (invalidReq) {
     res.status(400);
     res.json({ message: 'Bad Request. Answer must have a body.' });
   } else {
-    var answers = currQuestion[0].answers;
-    var newId = answers[answers.length - 1].id + 1;
-    var timeNow = new Date();
-    var timeStr = timeNow.toUTCString();
+  	const answers = currQuestion[0].answers;
+    const newId = answers[answers.length - 1].id + 1;
+    const timeNow = new Date();
+    const timeStr = timeNow.toUTCString();
     answers.push({
       id: newId,
       body: reqBody.body,
       username: reqBody.username,
       timeAnswered: timeStr,
-      accepted: false
+      accepted: false,
     });
-    res.json({ message: 'New answer: ' + reqBody.body + ' added', question: '' + currQuestion[0].title });
+    res.status(201);
+    res.json({ message: `New answer: ${reqBody.body} added`, question: `${currQuestion[0].title}` });
```

Comment:
Nice. You should have your responses with the keys `status`, `message` and `data`(if needed). Also, I noticed that some of your lines are longer than 80 characters. Airbnb style guide is configured with 100 max length but you should set your line max length to 80 characters in your eslint config file. This will notify you of any line that exceeds 80 characters.